sirepo/template/srw.py

```diff
@@ -507,9 +509,19 @@ def get_application_data(data, **kwargs):
 def get_data_file(run_dir, model, frame, options):
     if options.suffix == _SRW_LOG_FILE:
         return template_common.text_data_file(_SRW_LOG_FILE, run_dir)
+    if options.suffix == _PROGRESS_LOG_DIR:
+        d = run_dir.join(_PROGRESS_LOG_DIR)
+        f = _get_min_mtime_file(pkio.sorted_glob(d.join("*.log"))).path
+        return template_common.text_data_file(f.basename, d)
     return get_filename_for_model(model)
 
 
+def _get_min_mtime_file(files):
```

Comment:
Is this min_mtime or max_mtime? Instead of naming it by the algorithm, you should name it by the "what": most_recent.

Reply:
Oops, its supposed to be max, but I will change to most_recent